src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -17,22 +17,30 @@ namespace Microsoft.Docs.Build
 {
     internal class OpsConfigAdapter : IDisposable
     {
-        public static string ValidationServiceEndpoint =>
-            s_isProduction
-            ? "https://docs.microsoft.com"
-            : "https://ppe.docs.microsoft.com";
+        public static string ValidationServiceEndpoint => s_docsEnvironment switch
+        {
+            DocsEnvironment.Production => "docs.microsoft.com",
+            DocsEnvironment.Sandbox => "ppe.docs.microsoft.com",
+            DocsEnvironment.Internal => "ppe.docs.microsoft.com",
+            DocsEnvironment.Perf => "ppe.docs.microsoft.com",
+            _ => string.Empty
```

Comment:
nit: consider `throw` here because if we reached here, there must be some code bugs, better stop immediately.
```suggestion
            _ => throw InvalidOperationException(),
```